### DISCUSSION TITLE
symbol

### DISCUSSION THREAD

**Discussion by Imna1975** [Hekili/hekili#1342] [Q&A] [answered]
I have a question about the preview and I have a small head in the recommendation and can't assign it
![Unbenannt](https://user-images.githubusercontent.com/45740014/152029649-da5c06f7-8117-4f7d-91b3-197d3287ef16.jpg)

**Comment by Hekili** [comment, marked as answer]
It's probably a trinket.

If you haven't changed the default keybindings, you can pause the addon with `ALT-SHIFT-P` and mouseover the icon to see what it is.
